src/ctree-app/ctree-landing.html

```diff
@@ -53,6 +53,8 @@
 <link rel="import" href="../ctree-import/ctree-import.html">
 <link rel="import" href="../ctree-toggle-button/ctree-toggle-button.html">
 
+<link rel="import" href="../ctree-testimonials/ctree-testimonials.html">
```

Comment:
I like this being a separate component, but I'm not sure it needs its own folder.  It might make sense to have a ctree-landing folder, or just put it in ctree-app.  The way I like to think of folders is logical pieces which might be used separately.  Testimonials is probably something only useful for the cTree when it's in the form of a web app (a site hosting a bunch of cTrees), but many of the other components could be used if someone wanted to embed a cTree on their own site.

Reply:
Actually, after looking at this more I see that you made another component, so it makes sense to separate them both into the same folder.